mungegithub/github/github.go

```diff
@@ -1289,6 +1292,41 @@ func (obj *MungeObject) MergePR(who string) error {
 }
 
 // ListComments returns all comments for the issue/PR in question
+func (obj *MungeObject) ListPRComments() ([]github.PullRequestComment, error) {
```

Comment:
Unfortunately, ListComments doesn't return comments made on the diff of the pull-request, like this one.